docs/topics/publishing.md

```diff
@@ -152,3 +162,9 @@ Execute the commands in your local checked out repositories and push the tags us
 ```
 git push --tags
 ```
+
+## Announce new versions
+
+Announcing new releases should be done to:
+
+- external https://lists.wikimedia.org/mailman/listinfo/wikibaseug
```

Comment:
Don't we also post these to wikidata-tech?

Reply:
I don't know sounded like it was related to wikidata

Reply:
Tricky. The Wikibase UG is targeted more towards the community that these builds are ostensibly for, but isn't restricted to technology-related discussions.

Wikidata Tech is technology-focused, but (from the point of view of a third party) it's about "another _site_", rather than the Wikibase software; so not as relevant and less likely to be picked unless federating with or drawing from Wikidata.

(That's if they subscribe to any mailing list - it's more likely if a suggestion is given.)

Reply:
I suggest that we consult the "final" (it will be a subject to change over time likely) list of communication channels with WMDE Community Communication and adjust the text here before the ADR is approved.
For this pull request only listing the wikibase UG list should be sufficient and seen as a "placeholder".